lib/listen/adapter/darwin.rb

```diff
@@ -44,9 +44,9 @@ def _run
         require 'rb-fsevent'
         worker = FSEvent.new
         dirs_to_watch = @callbacks.keys.map(&:to_s)
-        _log(:info) { "fsevent: watching: #{dirs_to_watch.inspect}" }
+        Listen.logger.info { "fsevent: watching: #{dirs_to_watch.inspect}" }
         worker.watch(dirs_to_watch, { latency: options.latency }, &method(:_process_changes))
-        @worker_thread = Thread.new { _run_worker(worker) }
+        @worker_thread = Listen::Thread.new("worker_thread") { worker.run }
```

Comment:
Using the fully qualified name is not strictly necessary although it is possibly better since `Thread.new` might give the wrong impression.

Reply:
Yes, that was my thought. `Thread.new` would lead many astray.